gui/quantum.lua

```diff
@@ -275,80 +231,42 @@ local function order_minecart(pos)
     quickfort_orders.enqueue_additional_order(quickfort_ctx, 'wooden minecart')
     quickfort_orders.create_orders(quickfort_ctx)
 end
-
-local function create_quantum(pos, qsp_pos, feeder_tiles, name, trackstop_dir,
-                              allow_refuse)
-    local dsg = allow_refuse and 'yr' or 'c'
-    local stats = quickfort.apply_blueprint{mode='place', data=dsg, pos=qsp_pos}
-    if stats.place_designated.value == 0 then
-        error(('failed to place quantum stockpile at (%d, %d, %d)')
-              :format(qsp_pos.x, qsp_pos.y, qsp_pos.z))
+--Key Input
+function QuantumWin:onInput(keys)
+    pos = dfhack.gui.getSelectedStockpile()
```

Comment:
leftover line? it doesn't seem to do anything other than set an unused global variable